Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml

```diff
@@ -661,77 +661,11 @@
         <glowRadius>12</glowRadius>
         <glowColor>(217,217,208,0)</glowColor>
       </li>
-    </comps>
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="StandingLampBase">
-    <defName>StandingLamp_Red</defName>
-    <label>standing lamp (red)</label>
```

Comment:
Yep, never touch the Core :stuck_out_tongue: 

So will you just re-add the lines you deleted and send me another pull request? Or should I do it? The way it is now is no good.